packages/rollup/src/index.js

```diff
@@ -39,10 +39,10 @@ try {
 // Store the cache forever to re-use on each build
 let cacheMap = Object.create(null);
 
-// Generate a unique cache ID based on the input/outputOptions
-function computeCacheKey(inputOptions, outputOptions) {
+// Generate a unique cache ID based on the given json data
+function computeCacheKey(cacheKeyData) {
```

Comment:
interesting, it took two args before but we only passed one?

Reply:
That was me playing around with how the cache key was calculated and not cleaning up after myself.

Originally the key was basically all input/outputOptions. I think that's overkill and basically just the entry-files is enough.

Note that two keys conflicting isn't the end of the world, it just means the cache data passed to rollup might be useless (then get overwritten and useless for the other rollup rule with the conflicting key).